configs/nas/mmcls/autoformer/autoformer_supernet_8xb256_in1k.py

```diff
@@ -17,12 +17,13 @@
     _scope_='mmrazor',
     type='SearchableImageClassifier',
     data_preprocessor=data_preprocessor,
-    backbone=dict(type='AutoformerBackbone'),
+    backbone=dict(_scope_='mmrazor', type='AutoformerBackbone'),
     neck=None,
     head=dict(
+        _scope_='mmrazor',
         type='DynamicLinearClsHead',
         num_classes=1000,
-        in_channels=640,
+        in_channels=624,
```

Comment:
The `in_channels` of SuperNet should be 640, because when the head dim is 64 and the number of head is 10, the max embed dim would be 10 * 64 = 640.